android/databases/build.gradle

```diff
@@ -0,0 +1,94 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'kotlin-kapt'
+
+android {
+    compileSdkVersion 28
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    defaultConfig {
+        minSdkVersion 23
+        targetSdkVersion 28
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+configurations {
+    ktlint
+}
+
+dependencies {
+    ext.kotlinVersion = "1.3.31"
+    ext.rxAndroidVersion = "2.1.0"
+    ext.rxKotlinVersion = "2.3.0"
+    ext.okhttp3Version = "3.12.0"
```

Comment:
Let's move the versions into the root `android/build.gradle` so the versions can be shared between modules

Reply:
fixed